packages/appsync-emulator-serverless/lambdaSource.js

```diff
@@ -19,8 +19,16 @@ const lambdaSource = async (
     throw new Error(`Cannot find function config for function : ${fn}`);
   }
 
+  // Default to empty string, path.join will resolve this automatically
+  let modulePrefix = '';
```

Comment:
Curious what you think about calling this `buildPrefix`?

Reply:
Don't see any problems with this, will change.